README.md

```diff
@@ -0,0 +1,69 @@
+# LAB - Class 32
+
+## Project: Django REST Framework Authentication & Production Server
+
+### Author: Katrina Hill
+
+- attribution to instructor JB Tellez for class demo code modified for this application
+
+### Setup
+
+- PORT - <http://0.0.0.0:8000/>
+- DATABASE_URL - <http://0.0.0.0:8000/api/v1/food/>
+
+### How to initialize/run the application
+
+#### Terminal 1
+
+- create virtual environment: python3 -venv .venv
+- activate virtual environment: source .venv/bin/activate
+- pip install pytest
+- pip freeze > requirements.txt
```

Comment:
I think this can be omitted. `pip freeze` captures the packages that the creator/editor has been using in their environment into requirements.txt. For a user (or container) using requirements.txt, they want to know how to recreate that creator's/editor's environment. The user can get that information from requirements txt.

For example, when I'm running other people's code, I'll turn on a blank virtual environment (`source .venv/bin/active`) and then install the packages listed in requirements.txt (`pip install -r requirements.txt`). The Dockerfile does something similar when creating the container image: `RUN pip install -r requirements.txt` on line 6.